packages/neuron-wallet/src/channel/index.ts

```diff
@@ -70,18 +71,91 @@ export class Listeners {
     })
   }
 
+  static checkPassword = (walletID: string, password: string) => {
+    const myWallet = wallets().find(wallet => wallet.id === walletID)
+    if (!myWallet) {
+      return {
+        status: ResponseStatus.Success,
+        result: false,
+        msg: 'Wallet not find',
+      }
+    }
+    if (myWallet.password === password) {
```

Comment:
Password should not be stored directly.

Reply:
It`s mock data.It will be encryted in keystore when the keystore is determined

Reply:
The mock is improper. Use a function like `verifyPassword` to hide the implementation. 

And the comparison should be something like `if (unlock(wallet, password).isUnlock)` because that the `myWallet.password` will never be there.